components/grid/loading-animation.md

```diff
@@ -10,66 +10,105 @@ position: 90
 
 # Loading Animation
 
-The loading animation indicates a data operation that requires more than 600ms to complete. The indicator appears as a loading sign over the Blazor Data Grid. The loading animation improves user experience with a visual hint that the requested action is still executing. The feature can prevent repetitive user actions.
+The Grid can show a loading animation to indicate data operations that take more than 600ms to complete. The indicator appears as a loading sign over the Blazor Data Grid. The loading animation improves user experience with a visual hint that the requested action is still executing. The feature can prevent repetitive user actions.
 
-The data operations that trigger the loading animation include:
+## Basics
+
+The Grid `EnableLoaderContainer` parameter determines if the component will show a built-in LoaderContainer for long operations. The loading animation is enabled by detault. The data operations that trigger the loading animation include:
 
 * [Paging]({%slug components/grid/features/paging%})
 * [Filtering]({%slug components/grid/filtering%})
 * [Sorting]({%slug components/grid/features/sorting%})
 * [Grouping]({%slug components/grid/features/grouping%}) 
 * [Expanding groups with load-on-demand]({%slug grid-group-lod%})
-* [Editing]({%slug components/grid/editing/overview%})
-* [Inserting]({%slug components/grid/editing/overview%})
-* [Deleting records]({%slug components/grid/editing/overview%})
+* [Creating, deleting or editing records]({%slug components/grid/editing/overview%})
+
+## Show LoaderContainer on Initial Load
+
+The Grid will not display a loading animation during its initial rendering and data load. The component cannot know when or even if data will be provided to it, especially when using the Grid `Data` parameter. Initial automatic loading sign can either show indefinitely, or it could prevent the user from altering any saved Grid state (such as changing filters). If you want to display a loading animation on initial load, you can use a [LoaderContainer component]({%slug loadercontainer-overview%}). See the example below or the [Grid Loading Animation Live Demo](https://demos.telerik.com/blazor-ui/grid/loading-animation).
```

Comment:
```suggestion
The Grid does not display a loading animation during its initial rendering and data load. The component cannot know when or even if data will be provided to it, especially when using the Grid `Data` parameter. An initial automatic loading sign can either show indefinitely, or it could prevent the user from altering any saved Grid state (such as changing filters).

If you want to display a loading animation on initial load, you can use a [LoaderContainer component]({%slug loadercontainer-overview%}). See the example below or the [Grid Loading Animation Live Demo](https://demos.telerik.com/blazor-ui/grid/loading-animation).
```